libs/viewer/include/viewer/SimpleViewer.h

```diff
@@ -243,6 +243,7 @@ class UTILS_PUBLIC SimpleViewer {
     Settings mSettings;
     int mSidebarWidth;
     uint32_t mFlags;
+    utils::Entity mMorphEntity;
```

Comment:
I don't understand what this is.  I think it's the first morphable entity in the asset?  What if the user wants to play with morphing on an entity other than the first one?

Reply:
Yeah, I think this just assumes that. 
I think it's still useful for testing morphing though. It's not a great UI. Do we want a list of all morphable entities? Maybe this can be done later?

Reply:
Another idea is to move this UI into the existing Hierarchy section, and simply parent the sliders under the node that they are associated with.

In the short term this could be useful for testing, but we could at least rename the variable.

Reply:
That's good idea. It's chagned.